_format/1.1/index.md

```diff
@@ -1834,8 +1909,575 @@ An error object **MAY** have the following members:
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "links": {
+      "self": "http://example.com/articles/1"
+    },
+    "data": {
+      "type": "articles",
+      "id": "1",
+      "attributes": {
+        "title": "JSON API paints my bikeshed!"
+      }
+    }
+  }]
+}
+```
+
+An operation that fetches a resource collection **MUST** target that collection
+either through the request endpoint or the operation's `ref` object.
+
+For example:
+
+```json
+PATCH /bulk HTTP/1.1
+Host: example.org
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "op": "get",
+    "ref": {
+      "type": "articles"
+    }
+  }]
+}
+```
+
+An example response:
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "links": {
+      "self": "http://example.com/articles"
+    },
+    "data": [{
+      "type": "articles",
+      "id": "1",
+      "attributes": {
+        "title": "JSON API paints my bikeshed!"
+      }
+    }, {
+      "type": "articles",
+      "id": "2",
+      "attributes": {
+        "title": "Rails is Omakase"
+      }
+    }]
+  }]
+}
+```
+
+#### Fetching Relationships
+
+An operation that fetches relationship data **MUST** target a relationship
+either through the request endpoint or the operation's `ref` object. For
+example:
+
+```json
+PATCH /bulk HTTP/1.1
+Host: example.org
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "op": "get",
+    "ref": {
+      "type": "articles",
+      "id": "1",
+      "relationship": "author"
+    }
+  }]
+}
+```
+
+An example response:
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "links": {
+      "self": "http://example.com/articles/1/relationships/author"
+    },
+    "data": {
+      "type": "people",
+      "id": "9"
+    }
+  }]
+}
+```
+
+### Creating Resources
+
+An operation that creates a resource **MUST** target a resource collection
+either through the request endpoint or the operation's `ref` object. The
+operation **MUST** include an `op` code of `"add"`.
+
+For example:
+
+```json
+PATCH /bulk HTTP/1.1
+Host: example.org
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "op": "add",
+    "data": {
+      "type": "articles",
+      "attributes": {
+        "title": "JSON API paints my bikeshed!"
+      }
+    }
+  }]
+}
+```
+
+An example response:
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "operations": [{
+    "links": {
+      "self": "http://example.com/articles/13"
+    },
+    "data": {
+      "type": "articles",
+      "id": "13",
+      "attributes": {
+        "title": "JSON API paints my bikeshed!"
+      }
+    }
+  }]
+}
+```
+
+### Updating Resources
+
+An operation that updates a resource **MUST** target that resource
+either through the request endpoint or the operation's `ref` object. The
+operation **MUST** include an `op` code of `"replace"`.
```

Comment:
Same as above comment - this targets via properties of `data`, but it's stated that ops must use `ref`.

Reply:
@courajs again - good catch!